protocol/avdl/keybase1/teamsearch.avdl

```diff
@@ -0,0 +1,18 @@
+@namespace("keybase.1")
+
+protocol teamSearch {
+    import idl "common.avdl";
+
+    record TeamSearchItem {
+        TeamID id;
+        string name;
+        union { null, string } description;
+        int memberCount;
+    }
+
+    record TeamSearchRes {
+        array<TeamSearchItem> results;
+    }
+
+    TeamSearchRes search(string query, int limit);
```

Comment:
should we do paging here like the featuredbots?

Reply:
The consensus in the meeting was that we shouldn't do that, because the number of results we show in the sidebar will always be like 3 or less.

I guess in the future if we ever want a more full-featured search screen dedicated to open teams, we could add a second protocol method for that maybe.

Reply:
I see, sounds good

Reply:
Yeah I didn't want to overcomplicate this, I think it is fine now to just return a small number of results.